products/impulse/README.md

```diff
@@ -0,0 +1,91 @@
+# helpwave impulse
+
+This document includes all the necessary information to understand the idea and concepts behind helpwave impulse. 
+
+## The idea
+
+The aim of helpwave impulse is to reduce the load on the health care sector by helping people to live a healthier lifestyle. This will reduce the occurrence of illness, which leads to a more sustainable population. Healthier in this context means an improvement of the nutrition, fitness or mental health.
+
+## The concept
+
+### Understanding the individual needs
+
+A short Questionnaire should used to evaluate the needs of a healthier lifestyle for a user. This is import because each user has an individual lifestyle which creates the problem to choose the right task to great a healthier baseline. The Questions should evaluate the needs in the dimension’s nutrition, fitness and mental health. A user should not be encouraged to appropriate bad behaviour related to their lifestyle with this app.
+
+In the following context this information is referred to as lifestyle profile of a user.
+
+#### Nutrition
+
+As a reference for the Questions for the nutrition[^Nutrition] the guidelines of the DGE should be followed.
+
+#### Fitness
+
+The physical activity level (PAL[^PAL]) should be used to evaluate the activity level of a user. By knowing this a suitable baseline of activity could be inferred for a user.
+
+| PAL-Value | activity level                                           | example                                          |
+|-----------|----------------------------------------------------------|--------------------------------------------------|
+| 1,2 - 1,3 | sedentary or lying lifestyle                             | Immobile, bedridden people                       |
+| 1,4 - 1,5 | sedentary work, little strenuous leisure time            | Office worker                                    |
+| 1,6 - 1,7 | Sitting, short-term walking and standing work            | student                                          |
+| 1,8 - 1,9 | Mostly walking and standing work                         | Salesperson, waiter, craftsman                   |
+| 2,0 - 2,4 | Physically demanding job or very active leisure activity | Construction worker, farmer, competitive athlete |
+
+#### Mental health
+
+> Disclaimer: This app will not replace a professional psychologist.
+
+The app should evaluate the mental condition of a user. This is important so that the app isn’t overwhelming the user with the baseline tasks. 
+
+### Baseline Tasks (Daily's)
+
+The baseline tasks are tasks that should be done every day to improve the lifestyle of a user. These tasks should be derived from the information which is given be the questionnaire. So that only relevant task that would improve the lifestyle would be chosen.
```

Comment:
The baseline tasks are tasks that should be done every day to improve the lifestyle of a user. These tasks should be derived from the information which is given be the questionnaire. So that only relevant tasks will be chosen for the user.